docs/adding-env-variables.md

```diff
@@ -0,0 +1,73 @@
+# Adding Environment Variables
+
+Environment variables are a great way to molding your developer environment your own. You can create alias's or variables that you can refer to in your command line. In our case, it's useful for configuring `create-react-app` to work with our Sass modules. To permanently set your `SASS_PATH` environment follow the instructions below:
+
+## Windows
+
+#### Using SETX
+
+Windows is straightforward and requires just one command.
+
+Open command line terminal, and enter the following:
+
+```bat
+SETX SASS_PATH ./node_modules
+```
+
+> _NOTE:_ Be sure to close that terminal and open a new one for the new settings to take effect.
+
+## Mac OSx
+
+#### Open your `bash_profile`
+You can add an environment variable to your system by updating your `~/.bash_profile`. Open a terminal window and follow these steps:
+
+```sh
+touch ~/.bash_profile # run this line only if the file doesn't exist
```

Comment:
There's not really any reason to do this since the next command will create it if it doesn't exist anyway

Reply:
true...is that also true of `nano`?